source/about/staff.rst

```diff
@@ -12,14 +12,20 @@ The **Project Leaders** are the stewards of the Sponge project, and make the fin
 +-----------------------------------+-----------------------------------+
 | Name                              | Role                              |
 +===================================+===================================+
-| blood                             | Implementation Development Lead   |
```

Comment:
@jamierocks 

I lead the Vanilla impl and API. Vanilla has become somewhat of my baby to work on :P.